src/backend/src/CMakeLists.txt

```diff
@@ -6,19 +6,23 @@ include(cmake/compile_proto.cmake)
 
 foreach(COMPONENT_NAME ${COMPONENTS_LIST})
     compile_proto_pb(${COMPONENT_NAME} PB_COMPILED_SOURCE)
-    list(APPEND PB_COMPILED_SOURCES ${PB_COMPILED_SOURCE})
-
     compile_proto_grpc(${COMPONENT_NAME} GRPC_COMPILED_SOURCE)
-    list(APPEND GRPC_COMPILED_SOURCES ${GRPC_COMPILED_SOURCE})
+    add_library(${COMPONENT_NAME}_proto_gens ${GRPC_COMPILED_SOURCE} ${PB_COMPILED_SOURCE})
```

Comment:
But then that means that we create a new library for each plugin, right? `libaction_proto_gens.so`, `libtelemetry_proto_gens.so`, ...

Reply:
And if yes, should we always generate them as static archives? It feels a bit weird to me to have a shared lib just for the proto file (but maybe it's just me :sweat_smile:)

Reply:
Sg, made them static